README.md

```diff
@@ -2,9 +2,30 @@
 
 # cluster-api-provider-cloud-director-app
 
-**Source**: https://github.com/vmware/cluster-api-provider-cloud-director/tree/main/config/default
-**Commit**: 003336f85f44312252553dc1369faf409e18c10b
+## Source
 
-* Look up FILECHECK in this repo to find code / files to double check.
-* Manual chart creation (not kustomize generated).
-* Basic crd creation check in values.yaml.
+The image is currently built manually from a specific commit since the only release is outdated and `main` is the development branch. The capvcd helm chart is based on the output of `kustomize build` in the following folder:
+
+* Repo: https://github.com/vmware/cluster-api-provider-cloud-director/tree/main/config/default
+* Commit: 003336f85f44312252553dc1369faf409e18c10b
+
+## How to use
+
+There are no credentials in the CAPVCD controller. The credentials are stored in a secret in a namespace that is referenced in the cluster chart.
+
+* Install cluster API v1.1.3.
+
+`clusterctl init --core cluster-api:v1.1.3 -b kubeadm:v1.1.3 -c kubeadm:v1.1.3`
```

Comment:
Do you plan to keep this version up to date?

Reply:
I'm putting it in the README for now as I'm the only tinkering with it but we'll make it more generic later on. The goal is to be consistent across providers (on-prem at least) down the line.